src/constants.ts

```diff
@@ -0,0 +1,7 @@
+import { Extension } from '@colony/colony-js';
+
+export const SUPPORTED_EXTENSION_IDS = [
+  Extension.OneTxPayment,
+  Extension.VotingReputation,
+];
+export const INITIALISABLE_EXTENSION_IDS = [Extension.VotingReputation];
```

Comment:
I separated supported ids (meaning all the extensions we want to track) from initialisable ids (for which we want to add a listener for ExtensionInitialised event)